main.js

```diff
@@ -348,4 +348,35 @@ function validateEmail() {
 contForm.addEventListener('submit', (e) => {
   e.preventDefault();
   validateEmail();
-});
\ No newline at end of file
+});
+
+const myForm = document.forms.contactForm;
+const nameInput = myForm.fullName;
+const emailInput = myForm.emailAddress;
+const messageInput = myForm.message;
+
+const formData = {
+  fullName: '',
+  emailAddress: '',
+  message: '',
+};
+
+const setData = () => {
+  localStorage.setItem('formData', JSON.stringify(formData));
+};
+
+myForm.addEventListener('change', () => {
+  formData.fullName = nameInput.value;
+  formData.email = emailInput.value;
+  formData.message = messageInput.value;
+  setData();
+});
+
+const getData = () => {
+  const localFormData = JSON.parse(localStorage.getItem('formData'));
+  nameInput.value = localFormData.fullname;
+  emailInput.value = localFormData.email;
+  messageInput.value = localFormData.message;
+};
```

Comment:
- [ ] Good job storing the data on the localStorage! However, there's an error with the name field after loading the page:

When the user fills the form, the data is stored properly:
![Captura de pantalla 2022-04-16 a la(s) 4 20 37 p  m](https://user-images.githubusercontent.com/83717285/163691799-666d4157-5528-46c8-ab5a-d7d4db59f059.png)

After reload, the name value is `undefined`:
![Captura de pantalla 2022-04-16 a la(s) 4 20 53 p  m](https://user-images.githubusercontent.com/83717285/163691809-54731e8c-9912-45af-a1db-510231055a30.png)

- Please fix the name field value error in order to retrieve all data properly.

Tip: Make sure there are no typos 😉